### ISSUE TITLE
[Effect Suggestion] Remove all ammo from the player's weapons

### ISSUE THREAD

**Issue by TheCrazyAussie4** [gta-chaos-mod/ChaosModV#2904]
Remove all ammo from every weapon/Drain all ammo from every weapon/ No more ammo for you
Drain all ammo from every weapon
**Duration (choose one and replace the corresponding "[ ]" with "[X]")**
- [ ] Long
- [ ] Short
- [X] No duration

**Effect Description**
It removes all ammo from every weapon the player has, so ped's still have ammo.

**Comment by TheCrazyAussie4** [comment]
it's similar to this: https://github.com/gta-chaos-mod/ChaosModV/issues/441, but I thought it was different enough to constitute its own post, please tell me if it's too similar.

**Comment by AshtonGoBoom** [comment]
Wouldn't that basically just be a different version of remove all weapons?

**Comment by TheCrazyAussie4** [comment]
yeah, I guess, but it's more or less annoying, depending on how you look at it. you still have all the weapons you just can't use them.

**Comment by SeanMirrsen** [comment]
It's not functionally any different from "remove all weapons", since you can't really pick up 'ammo', only more weapons (except at stores I guess).

At best it could be folded into the same effect as an alternate. Either remove all weapons, or all ammo.
